.gitignore

```diff
@@ -18,6 +18,8 @@ cypress-a11y-report.json
 /frontend/@types/gql
 /frontend/gui_test_screenshots
 /frontend/packages/integration-tests-cypress/cypress-a11y-report.json
+/frontend/packages/dev-console/integration-tests/cypress-a11y-report.json
+/frontend/packages/knative-plugin/integration-tests/cypress-a11y-report.json
```

Comment:
Ah this change is what stopped my approval. Will need @spadgett to approve likely. There are few with global permissions.

Reply:
Perhaps we change this to just say:

```
/frontend/packages/**/cypress-a11y-report.json
```
I believe that works... we don't want to commit any of these reports I assume.

Reply:
updated